besticon/iconserver/server_test.go

```diff
@@ -188,6 +201,21 @@ func TestGetLetterIcon(t *testing.T) {
 	assertIntegerInInterval(t, 1500, 1800, w.Body.Len())
 }
 
+func TestGetLetterIconSVG(t *testing.T) {
+	req, err := http.NewRequest("GET", "/lettericons/M-144-EFC25D.svg", nil)
```

Comment:
Do we even want to allow for the `size` to be in the path for `.svg`? I'm thinking the fact the we allow it is kind of ok by Postel's law on the one hand but could lead to misunderstanding one the other (we're not using this).

Reply:
Sure, let's turn off support for size entirely. We're not generating those with SVG so no need to parse them. Give me a sec here